collect_sources.sh

```diff
@@ -49,15 +49,11 @@ EXCLUDE='realvnc-vnc|oracle-java8-jdk'
 # exclude Raspbian packages with missing source code
 EXCLUDE+='|nodered|wiringpi'
 # exclude binary-only RevolutionPi packages
-EXCLUDE+='|logi-rts|logiclab|piserial|procon-web-iot|teamviewer-revpi'
-# exclude binary-only RevolutionPi packages
-EXCLUDE+='|pimodbus-master|pimodbus-slave'
+EXCLUDE+='|logiclab|piserial|procon-web-iot|teamviewer-revpi|revpi-modbus'
```

Comment:
I think logiclab and procon-web-iot are also not part of buster.
@pavlic244?

Reply:
I removed both